.github/workflows/ci.yml

```diff
@@ -72,7 +72,7 @@ jobs:
           --health-timeout 5s
           --health-retries 5
       elasticsearch:
-        image: hypothesis/elasticsearch:latest
+        image: hypothesis/elasticsearch:elasticsearch7.10
```

Comment:
Rather than using a "latest" tag here, I opted to use one that identifies the major and minor version. This will make future migrations "opt-in" for this project.